web/src/main/java/org/apache/shiro/web/filter/authc/FormAuthenticationFilter.java

```diff
@@ -30,6 +31,39 @@
 import javax.servlet.ServletResponse;
 import javax.servlet.http.HttpServletRequest;
 
+/**
```

Comment:
Just like I said in #824 Please remove this extraction. It's not deserving of it's own class